setup.cfg

```
@@ -17,5 +17,5 @@
 [options.entry_points]
 cylc.configure = 
     configure = cylc.rose.rose:get_rose_vars
-cylc.rose_fileinstall =
+cylc.fileinstall =
     install = cylc.rose.rose:rose_fileinstall
```

Namespace the plugin's entry points to avoid possible conflicts:

```suggestion
    rose = cylc.rose.rose:rose_fileinstall
```